etc/config-template.toml

```diff
@@ -27,6 +27,21 @@
 ## Set it small are good to debug and tuning.
 # refresh-config-interval = "30s"
 
+[readpool]
+## (Experimental) Whether to use a single thread pool to serve all the read requests. If it is
+## set to `true`, the `readpool.storage` and `readpool.coprocessor` sections will be invalid and
+## be ignored automatically.
+# unify-read-pool = false
```

Comment:
I thought it should be enabled by default?

Reply:
Let's do it a bit later... I think it should be a later PR which sets the default value in code to true that sets it to true here.